src/GNDStk/XML/test/XML.test.cpp

```diff
@@ -0,0 +1,9 @@
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+SCENARIO("Testing GNDStk XML") {
+
```

Comment:
What's happening here?

Reply:
From the looks of it, not very much. :-D

This is another example of the fact that code constructs don't exist in a vacuum. Presumably, I originally put this in as a placeholder, because the NJOY structure wants something there. The GNDStk XML class is involved all over the place in other tests, though, because reading and writing those XMLs is needed for all manner of other things. I could do something here, but it would invariably overlap with some other test somewhere else - a test of XML read, or XML write, or of various data access functions, or of our convert() functions, or whatever.